tools/c7n_azure/c7n_azure/resources/access_control.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2018 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from c7n_azure.query import QueryResourceManager
```

Comment:
We should get in the habit of keeping our imports clean.  Typical python style guide would have you sort them and seperate into groups, so the c7n_azure bits together, and the c7n bits together.

In practice, if you just put your cursor in the imports and press `ctrl+alt+o` (for optimize imports) in PyCharm it comes out perfect according to the style guide https://www.python.org/dev/peps/pep-0008/#imports